siteapp/management/commands/first_run.py

```diff
@@ -34,37 +34,60 @@ def handle(self, *args, **options):
             print("The database is not initialized yet.")
             sys.exit(1)
 
-        # Create AppSources that we want.
-        if os.path.exists("/mnt/q-files-host"):
-            # For our docker image.
-            AppSource.objects.get_or_create(
-                slug="host",
-                defaults={
-                    "spec": { "type": "local", "path": "/mnt/q-files-host" }
-                }
-            )
-        # Second, for 0.9.x startpack
-        # We can use forward slashes because we are storing the path in the database
-        # and the path will be applied correctly to the operating OS.
-        qfiles_path = 'q-files/vendors/govready/govready-q-files-startpack/q-files'
-        if os.path.exists(qfiles_path):
-            # For 0.9.x+.
-            AppSource.objects.get_or_create(
-                slug="govready-q-files-startpack",
-                defaults={
-                    "spec": { "type": "local", "path": qfiles_path }
-                }
-            )
-            # Load the AppSource's assessments (apps) we want
-            # We will do some hard-coding here temporarily
-            created_appsource = AppSource.objects.get(slug="govready-q-files-startpack")
-            for appname in ["speedyssp", "System-Description-Demo",
-                            "PTA-Demo", "rules-of-behavior", "lightweight-ato", "lightweight-ato-800-171"]:
-                print("Adding appname '{}' from AppSource '{}' to catalog.".format(appname, created_appsource))
-                try:
-                    appver = created_appsource.add_app_to_catalog(appname)
-                except Exception as e:
-                    raise
+        # Create the default organization.
+        if not Organization.objects.all().exists() and not Organization.objects.filter(name="main").exists():
+            org, result = Organization.objects.get_or_create(name="main", slug="main")
```

Comment:
```suggestion
            org = Organization.objects.create(name="main", slug="main")
```
Dont need to get if you are doing the fitler check for the main organization and if any orgs exist.